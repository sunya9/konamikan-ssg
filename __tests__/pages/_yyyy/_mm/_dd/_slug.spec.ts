import { mount } from '@vue/test-utils'
import { post } from '@/__tests__/fixtures/postOrPage'
import Article from '@/pages/_yyyy/_mm/_dd/_slug.vue'

describe('article page', () => {
  it('normal', () => {
    expect(
      mount(Article, {
        mocks: {
          post
        }
      })
    ).toMatchSnapshot()
  })
})
