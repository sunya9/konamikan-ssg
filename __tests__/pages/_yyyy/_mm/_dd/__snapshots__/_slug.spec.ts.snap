// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`article page normal 1`] = `
<div>
  <header class="hero is-medium fix-background" style="background-image: url(cover.png);">
    <div class="hero-head">
      <nav class="navbar is-transparent">
        <div class="container">
          <div class="navbar-brand">
            <div class="navbar-brand"><a class="navbar-item has-text-white-bis">
                title
              </a></div> <a href="#" aria-label="Menu" class="navbar-burger has-text-white"><span aria-hidden="true"></span> <span aria-hidden="true"></span> <span aria-hidden="true"></span></a>
          </div>
          <div class="navbar-menu is-relative">
            <div class="navbar-end">
              <div class="navbar-item">
                <div class="control has-icons-right"><label for="search-form-input" class="is-sr-only">Search</label> <input id="search-form-input" type="search" placeholder="Search..." class="input is-small search"> <span class="icon is-small is-right"><svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search"><circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line></svg></span></div>
                <!---->
              </div> <a href="/rss" class="navbar-item has-text-white-ter"><span class="icon"><svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-rss"><path d="M4 11a9 9 0 0 1 9 9"></path><path d="M4 4a16 16 0 0 1 16 16"></path><circle cx="5" cy="19" r="1"></circle></svg></span> <span class="is-sr-only">RSS</span></a>
            </div>
          </div>
        </div>
      </nav>
    </div>
    <div class="hero-body fix-z-index">
      <div class="container has-text-centered">
        <h1 class="title is-1 has-text-white">
          title
        </h1>
        <h2 class="subtitle has-text-white-bis">

        </h2>
      </div>
    </div>
    <!---->
  </header>
  <main class="section">
    <div class="container">
      <article>
        <div class="content">
          <div>
            <p>article</p>
          </div>
        </div>
        <footer>
          <div class="tags"><a class="tag is-rounded is-medium">
              tag
            </a></div>
          <div class="columns">
            <div class="column is-half is-offset-one-quarter">
              <div class="box">
                <div class="media">
                  <div class="media-left">
                    <div class="image is-64x64"><a><img src="" width="128" height="128" class="is-rounded"></a></div>
                  </div>
                  <div class="media-content">
                    <div class="content is-small">
                      <div class="title is-6"><a>
                          Tester
                        </a></div>
                      <div class="subtitle is-6">
                        <!---->
                      </div>
                      <p></p>
                      <div class="level is-mobile">
                        <div class="level-left">
                          <!---->
                          <!---->
                          <!---->
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </footer>
      </article>
    </div>
  </main>
</div>
`;
